src/master-playlist-controller.js

```diff
@@ -831,40 +847,81 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     });
   }
 
-  updateCues_(media) {
+  updateCues_(media, offset = 0) {
     if (!this.useCueTags_ || !media.segments) {
       return;
     }
 
-    while (this.cueTagsTrack_.cues.length) {
-      this.cueTagsTrack_.removeCue(this.cueTagsTrack_.cues[0]);
```

Comment:
I see that we're never removing cues now. Are we OK with the assumption that different playlists will always have cues at the same segments and times (included in this is that segments have the same duration across playlists)?